src/components/AddFamily.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { useState } from 'react';
+import DropDownMenu from './DropDownMenu.js';
+
+/** AddFamily is a pop-up window that lets the user add a person as family
+ *  - Are all the state relevant when we get to backend? We'll have to see.
+ *  - the "submit" needs to be handled
+ *  - ListOfDuties should be mapped to a dropdown menu
+ */
```

Comment:
Nice. You can also add documentation to the AddFamily function visible when implementing it elsewhere.
Regardless, love that you have the excess energy for doing this.